lib/UI.js

```diff
@@ -185,6 +193,20 @@ define(function (require, exports, module) {
                 IframeBrowser.attachPreview();
             }
         });
+
+        Compatibility.supportsIFrameHTMLBlobURL(function(err, isCompatible) {
+            if(err) {
+                console.error("[Brackets IFrame-Browser] Unexpected error:", err);
+                return callback();
+            }
+
+            // If we are in IE v<11, we hide the detachable preview button
+            if(!isCompatible && document.all) {
+                $("#liveDevButton").css("display", "none");
```

Comment:
Let's `console.log` here to let people know we're turning off detachable preview in pre-IE 11.  Then it's easier to see why it isn't there, if you're debugging.